src/gui/src/assets/keyboard_mixin.js

```diff
@@ -83,8 +92,13 @@ const keyboardMixin = targetId => ({
             card.show = this.card_items[this.pos].querySelector(".card");
             card.id = this.card_items[this.pos].dataset.id;
```

Comment:
If nothing is focused dataset can be undefined, maybe just abor the function in that case or select the first card?

![image](https://user-images.githubusercontent.com/6696618/156178633-0d9dbded-a9bd-466b-9599-da3f257fd638.png)

Reply:
As I didn't change that line -> #78